Backend/Backend/Controllers/Responses/PodcastResponses.cs

```diff
@@ -37,9 +39,21 @@ public EpisodeResponse(Episode e, string domainUrl)
     public string AudioUrl { get; set; } = string.Empty;
     public string ThumbnailUrl { get; set; } = string.Empty;
     public int Likes { get; set; } = 0;
+    
     public List<CommentResponse> Comments { get; set; } = new();
 }
 
+[BindProperties]
+public class EpisodeResponse2 : EpisodeResponse
```

Comment:
I think adding the podcast name to the episode response class would make more sense.